### PR TITLE
added zecat button for React

### PR DESCRIPTION
**Pull request by damfinkel**
## Summary

Added Zecat buttons

## Screenshots

image

### CODE REVIEW

File: cookbook-react/src/recipes/buttons/zecat/styles.module.scss [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+$smalt: #052E91;
+$white: #FFF;
+$black: #000;
+$mercury: #E1E1E1;
+
+.button {
+  border-radius: 5px;
+  color: $white;
+  font-size: 16px;
+  min-width: 200px; // Just for presentational purposes, remove in real project
```
Comment (pabloferro) [inline cookbook-react/src/recipes/buttons/zecat/styles.module.scss:11]: Maybe we can move this to a different css file with a comment to be deleted, and apply its style to the container div

File: cookbook-react/src/recipes/buttons/zecat/styles.module.scss [hunk 2]
```diff
@@ -0,0 +1,41 @@
+$denim: #0B43D1;
+$smalt: #052E91;
+$white: #FFF;
+$black: #000;
+$mercury: #E1E1E1;
```
Comment (pabloferro) [inline cookbook-react/src/recipes/buttons/zecat/styles.module.scss:5]: I's give them more meaningful names here like:
```suggestion
$backroundColor: #0B43D1;
$activeColor: #052E91;
$color: #FFF;
$hoverShadow: 0px 0px 4px 1px rgba($black, 0.2);
$disabledColor: #E1E1E1;
```

File: cookbook-react/src/recipes/buttons/zecat/styles.module.scss [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+    color: $denim;
+
+    &:disabled {
+      border: 1px solid $mercury;
+      color: $mercury;
+    }
+  }
+
+  &:hover:not(:disabled) {
+    // stylelint-disable-next-line length-zero-no-unit
```
Comment (pabloferro) [inline cookbook-react/src/recipes/buttons/zecat/styles.module.scss:38]: What's the issue with this rule?

Comment (damfinkel) [reply cookbook-react/src/recipes/buttons/zecat/styles.module.scss]: I thought the 0.2 value for the alpha in rgba apparently bothers the linter, but now I just realized it was the "0px". Already fixed it, my bad

File: cookbook-react/src/recipes/buttons/zecat/styles.module.scss (hunk 1 above)
Comment (damfinkel) [reply cookbook-react/src/recipes/buttons/zecat/styles.module.scss:11]: I don't know, I don't think this is a problem. Each project will remove this, or else they'll have fixed button sizes.

File: cookbook-react/src/recipes/buttons/zecat/index.tsx [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+    <div className="column center">
+      <button type="button" className={`${styles.button} ${styles.primary} m-bottom-2`}>Primary</button>
+      <button type="button" className={`${styles.button} ${styles.primary} m-bottom-2`} disabled>Primary Disabled</button>
+      <button type="button" className={`${styles.button} ${styles.secondary} m-bottom-2`}>Secondary</button>
+      <button type="button" className={`${styles.button} ${styles.secondary}`} disabled>Secondary Disabled</button>
+    </div>    
+  )
+}
+
+export default ButtonExample;
```
Comment (pabloferro) [inline cookbook-react/src/recipes/buttons/zecat/index.tsx:21]: Add new line 😅

**Review by pabloferro — CHANGES_REQUESTED** [review]

**Review by pabloferro — APPROVED** [review]

**Pull request merged**